tests/test_translate_json.py

```diff
@@ -1,54 +1,47 @@
-        self.assertEqual(self.original_json['theme'], "default")
-        self.assertEqual(self.original_json['schema_version'], "0.0.1")
-        self.assertEqual(self.original_json['data_version'], "0.0.1")
-        self.assertEqual(self.original_json['legal_basis'], "StatisticsOfTradeAct")
-        self.assertEqual(self.original_json['description'], "mbs_0106")
-
-    def test_translated_title_data(self):
-        """Test that title data we want / do not want to be translated is there after translation"""
-        self.assertEqual(self.translated_json['title'], "Arolwg Busnes Misol")
-        self.assertEqual(self.translated_json['survey_id'], "009")
-        self.assertEqual(self.translated_json['legal_basis'], "StatisticsOfTradeAct")
-        self.assertEqual(self.translated_json['description'], "mbs_0106")
-
-    def test_translation_of_title_data(self):
-        """When original json gets passed through function, correct title data should translate"""
-        container = deepcopy(self.original_json)
-        self.assertEqual(container['title'], "Monthly Business Survey")
-        translate_survey(container, translation_spreadsheet)
-        self.assertEqual(container['title'], "Arolwg Busnes Misol")
-
-    def test_translation_of_section_data(self):
-        """When original json gets passed through function, correct section data should translate"""
-        container = deepcopy(self.original_json)
-        self.assertEqual(container['sections'][0]['title'], "Navigation Title")
-        translate_survey(container, translation_spreadsheet)
-        self.assertEqual(container['sections'][0]['title'], "Teitl Llywio")
-
-    def test_translation_of_group_data(self):
-        """When original json gets passed through function, correct group data should translate"""
-        container = deepcopy(self.original_json)
-        self.assertEqual(container['sections'][1]['groups'][0]['title'], "Summary")
-        translate_survey(container, translation_spreadsheet)
-        self.assertEqual(container['sections'][1]['groups'][0]['title'], "Crynodeb")
+
+    def test_extract_translation_data(self):
+        json_file = "../tests/schemas/test_translation.json"
+        output_directory = "../tests/pot-files"
+        run_extract(json_file, output_directory)
+
+        line_count = 0
+        msgid_count = 0
+        pot_file = "../tests/pot-files/test_translation.pot"
+        with open(pot_file) as f:
+            for line in f:
+                line_count += 1
+                if line.startswith('msgid'):
+                    msgid_count += 1
+
+        assert(line_count == 332)
+        assert(msgid_count == 52)
+
+    def test_translate_survey(self):
+        translation_file = "../tests/po-files/cy/test_translation.po"
+        json_file = "../tests/schemas/test_translation.json"
+        output_directory = "../tests/schemas/cy/"
+        number_of_translations, missing_translations = run_translate(translation_file, json_file, output_directory)
```

Comment:
Looks like the order of these arguments now needs updating due to changes in run_translate